### DISCUSSION TITLE
physx/buildtools/presets/public/linux.user.xml not found

### DISCUSSION THREAD

**Discussion by 123uiui** [NVIDIA-Omniverse/PhysX#335] [Q&A]
I want to compile PhysX on Ubuntu 22，b
![屏幕截图 2024-11-25 135031](https://github.com/user-attachments/assets/5e168a0b-7647-4dda-9f6f-f67e370c9489)
ut I encountered an error.

**Comment by vreutskyy** [comment]
Hi @123uiui 
Could you try to build it under a different username? A name that doesn't have the word 'user' in it :)
There may be a bug in the build script.
Thanks

**Reply by 123uiui** [reply]
I changed the username to "zxc," and the script executed normally.

**Reply by vreutskyy** [reply]
Thanks
